HW_3/HW_3/task_2.py

```diff
@@ -0,0 +1,25 @@
+"""
+Задайте список целых чисел. Верните список с произведениями его парных элементов.
+Парой считаются первый и последний элемент, второй и предпоследний и т.д.
+Если элементов нечетное количество – центральный элемент умножается сам на себя.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <list>
+
+Пример:
+[2, 3, 4, 5, 6]
+[12, 15, 16]
+
+[2, 3, 5, 6]
+[12, 15]
+"""
+
+def mult_lst(lst):
+    l = len(lst)//2 + 1 if len(lst) % 2 != 0 else len(lst)//2
+    new_lst = [lst[i]*lst[len(lst)-i-1] for i in range(l)]
```

Comment:
Можно и так:
```suggestion
    l = (len(lst) + 1) // 2
    new_lst = [lst[i] * lst[-i - 1] for i in range(l)]
```
К тому же, Наталья, пожалуйста, не забывайте про PEP8.